compiler/rustc_hir/src/definitions.rs

```diff
@@ -377,22 +377,12 @@ impl Definitions {
     }
 
     #[inline(always)]
-    pub fn local_def_path_hash_to_def_id(
-        &self,
-        hash: DefPathHash,
-        err_msg: &dyn std::fmt::Debug,
-    ) -> LocalDefId {
+    pub fn local_def_path_hash_to_def_id(&self, hash: DefPathHash) -> Option<LocalDefId> {
```

Comment:
Please add a doc string:
```rust
/// Returns `None` if the `DefPathHash` does not correspond to a `LocalDefId` 
/// in the current compilation session. This can legitimately happen if the 
/// `DefPathHash` is from a `DefId` in an upstream crate or, during incr. comp.,
/// if the `DefPathHash` is from a previous compilation session and
/// the def-path does not exist anymore.
```

Reply:
Done.